### ISSUE TITLE
Plugin versions not upgraded

### ISSUE THREAD

**Issue by DanySK** [jmfayard/refreshVersions#179]
**Describe the bug**

Plugin support via `plugin.<pluginId>` works, but versions are not refreshed.

**Expected behavior**

For instance, if the previous version file is:
```
plugin.org.xtext.xtend=2.0.7

version.org.eclipse.xtend..org.eclipse.xtend.lib=2.19.0
```
by running `./gradlew refreshVersions` I'd expect (at the time of writing)

```
plugin.org.xtext.xtend=2.0.7
##                                   # available=2.0.8

version.org.eclipse.xtend..org.eclipse.xtend.lib=2.19.0
##                                   # available=2.21.0
```

I get instead:
```
plugin.org.xtext.xtend=2.0.7

version.org.eclipse.xtend..org.eclipse.xtend.lib=2.19.0
##                                   # available=2.21.0
```

**To Reproduce**

clone [this project at this exact commit](https://github.com/DanySK/listset/tree/83bbaba8389aba735356af0ababb671dee6f63a2)

run `./gradlew refreshVersions`

Note that several plugins could be upgraded (according to a search in https://plugins.gradle.org/) but no update proposal is provided by refreshVersions.

**Comment by DanySK** [comment]
Version 0.9.5 fixes this issues -- also, it looks like it accumulated quite a number of upgrades on that project :rofl: :

```
## suppress inspection "SpellCheckingInspection" for whole file
## suppress inspection "UnusedProperty" for whole file
##
## Dependencies and Plugin versions with their available updates
## Generated by $ ./gradlew refreshVersions
## Please, don't put extra comments in that file yet, keeping them is not supported yet.

plugin.com.github.spotbugs=4.0.5
##             # available=4.0.6
##             # available=4.0.7
##             # available=4.0.8
##             # available=4.1.0
##             # available=4.2.0
##             # available=4.2.1
##             # available=4.2.2
##             # available=4.2.3
##             # available=4.2.4
##             # available=4.3.0
##             # available=4.4.0
##             # available=4.4.1
##             # available=4.4.2
##             # available=4.4.3
##             # available=4.4.4
##             # available=4.4.5
##             # available=4.5.0

plugin.org.danilopianini.git-sensitive-semantic-versioning=0.1.0
##                                             # available=0.2.1-dev00+03a632e
##                                             # available=0.2.1-dev00+ceeb3fe
##                                             # available=0.2.1-dev00+3bf70f1
##                                             # available=0.2.1-dev00+a929cc4
##                                             # available=0.2.1-dev00+91bc4c6
##                                             # available=0.2.1-dev00+514baf8
##                                             # available=0.2.1-dev00+f2e41d8
##                                             # available=0.2.1-dev00+9f9a8d9
##                                             # available=0.2.1-dev00+428ac38
##                                             # available=0.2.1-dev00+c085c53
##                                             # available=0.2.1-dev00+d54ff87
##                                             # available=0.2.1-dev00+5284013
##                                             # available=0.2.1
##                                             # available=0.2.2

plugin.org.danilopianini.javadoc.io-linker=0.1.4

plugin.org.danilopianini.publish-on-central=0.2.0
##                              # available=0.2.1-dev01+86ce856
##                              # available=0.2.1
##                              # available=0.2.2-dev01+82f26b0
##                              # available=0.2.2
##                              # available=0.2.3

plugin.org.jlleitschuh.gradle.ktlint=9.0.0
##                       # available=9.1.0
##                       # available=9.1.1
##                       # available=9.2.0
##                       # available=9.2.1
##                       # available=9.3.0

plugin.org.xtext.xtend=2.0.8

version.com.google.guava..guava=29.0-jre

version.junit.junit=4.13

version.org.apache.commons..commons-collections4=4.4

version.org.eclipse.xtend..org.eclipse.xtend.lib=2.23.0.M2

```

Is the behaviour (pick all versions, only the latest, exclude betas or versions with some regex) configurable yet?

**Comment by LouisCAD** [comment]
The release notes don't mention that it is configurable, so it's not.
You'll see it there once it is.
